libsyclinterface/helper/source/dpctl_error_handlers.cpp

```diff
@@ -98,36 +76,34 @@ void error_handler(const std::exception &e,
     int requested_level = requested_verbosity_level();
     int error_level = static_cast<int>(error_type);
 
-#ifdef ENABLE_GLOG
-    LOG_IF(ERROR, requested_level >= error_level)
-        << e.what() << " in " << func_name << " at " << file_name << ":"
-        << line_num << std::endl;
-#else
-    if (requested_level >= error_level) {
-        std::cerr << e.what() << " in " << func_name << " at " << file_name
-                  << ":" << line_num << std::endl;
+    std::stringstream ss;
```

Comment:
There is no need to create `stringstream` and `string` from it if condition `requested_level >= error_level` is not met.